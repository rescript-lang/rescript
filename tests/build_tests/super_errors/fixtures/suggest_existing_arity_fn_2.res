Console.log2(1)
