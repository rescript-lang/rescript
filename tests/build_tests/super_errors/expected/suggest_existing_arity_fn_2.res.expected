
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/suggest_existing_arity_fn_2.res[0m:[2m1:1-12[0m

  [1;31m1[0m [2mâ”‚[0m [1;31mConsole.log2[0m(1)
  2 [2mâ”‚[0m 

  This function call is incorrect.
  The function has type:
  (int, 'a) => unit
  
  - The function takes [1;33m2[0m unlabelled arguments, but is called with [1;31mjust 1[0m
  
  Hint: Try [1;33mConsole.log[0m instead (takes [1;33m1[0m argument).